scripts/zkai_attention_kv_snark_statement_receipt_gate.py

```
@@ -0,0 +1,880 @@
+    cases = require_list(payload["cases"], "cases")
+    expect_equal(len(cases), len(EXPECTED_MUTATION_INVENTORY), "case count")
+    expect_equal(payload["case_count"], len(cases), "case_count")
+    if not payload["all_mutations_rejected"] or not all(case.get("rejected") is True for case in cases):
+        raise AttentionKvSnarkReceiptError("not all SNARK receipt mutations rejected", layer="mutation_suite")
+    by_name = {case.get("mutation"): case for case in cases}
+    expect_equal(set(by_name), EXPECTED_MUTATION_SET, "case mutation set")
+    for mutation, surface in EXPECTED_MUTATION_INVENTORY:
+        case = by_name[mutation]
+        expect_equal(case.get("surface"), surface, f"surface for {mutation}")
+        if case.get("mutated_accepted") is True:
+            raise AttentionKvSnarkReceiptError(f"mutation accepted: {mutation}", layer="mutation_suite")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)